src/views/credits/credits.jsx

```diff
@@ -257,6 +257,7 @@ var Credits = React.createClass({
                 <p>
                     <a href="http://www.nsf.gov/">National Science Foundation</a>,
                     <a href="http://www.scratchfoundation.org/"> Scratch Foundation</a>,
+                    <a href="https://http://www.siegelendowment.org"> Siegel Family Endowment </a>
```

Comment:
Add a comma after this line, remove the space before the `</a>`, change `https://http://` to `https://`, and you should be good to go!

Reply:
Thanks.